modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -287,7 +287,9 @@ function ding_webtrekk_form_ding_loan_loans_form_alter(&$form, &$form_state) {
       // therefore we can't get a well ID to display. We'll note that it's an
       // ILL and use the title instead.
       else {
-        $material_ids[$loan->id] = 'Interlibrary Loan: ' . $loan->display_name;
+        $material_ids[$loan->id] = t('Interlibrary Loan: %title', [
```

Comment:
don't use the [ ] syntax - use array() instead

Reply:
Are you sure about that? From the README:

Reply:
det er hvad der står i [Code Guidelines](https://github.com/ding2/ding2/blob/master/docs/code_guidelines.md). Ellers så skal dén vist opdateres... ;-P

Reply:
You're right, but as I mentioned we do use bracket notation in a lot of other places and in new PR's they are often used.

Anyway, I updated the PR to use array() notation.